src/nvim/globals.h

```diff
@@ -459,6 +459,7 @@ EXTERN frame_T  *topframe;      /* top of the window frame tree */
  * one in the list, "curtab" is the current one.
  */
 EXTERN tabpage_T    *first_tabpage;
+EXTERN tabpage_T    *lasttab;
```

Comment:
this is a very misleading name when compared to `lastbuf`, `lastwin`, etc. `recent_tab` or `lastused_tab` is better. (I can't find any existing precedent.)

Reply:
Fixed--now it's `lastused_tab`.

Reply:
`prevwin` looks to serve this purpose for windows.  Would `prevtab` be better?

Reply:
It's now `prevtab`.  That name was used one other place and the variable used there has now been renamed `old_curtab` which is anyway consistent with other variables serving the same purpose (`old_curtab`) and still others serving an analogous purpose (`old_curwin`).